### PR TITLE
Update openthem_gw.markdown

### PR DESCRIPTION
**Pull request by Martidjen**
Updating opentherm_gw documentation for new feature in parent PR

## Proposed change

Dividing temperature precision into precision for current temperature and precision for setpoint temperature

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/47484
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/opentherm_gw.markdown [hunk 1]
```diff
@@ -60,12 +60,19 @@ The precision and floor_temperature settings that were supported in configuratio
...
+  type: float
+  default: "`0.5` for Celsius and `1.0` for Fahrenheit."
+Set Precision:
+  description: "The desired set precision for this device. Used as step size for setting temperature setpoint from the climate entity. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+  required: false
   type: float
   default: "`0.5` for Celsius and `1.0` for Fahrenheit."
 Floor Temperature:
   description: "Some thermostats round all temperatures down to the lower value according to their precision. Default behavior for Home Assistant is to round temperatures to the nearest value. Enable this setting to override this behavior and round to the lower value according to the configured precision."
+  required: false
```
Comment (mvn23) [inline source/_integrations/opentherm_gw.markdown]: Not related to the parent PR.

File: source/_integrations/opentherm_gw.markdown [hunk 2]
```diff
@@ -60,12 +60,19 @@ The precision and floor_temperature settings that were supported in configuratio
 The OpenTherm Gateway can be further configured through the integration settings in the web interface
 The following options are available:
 {% configuration %}
-Precision:
-  description: "The desired precision for this device. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+Read Precision:
+  description: "The desired read precision for this device. Used to display the current temperature on the climate entity. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+  required: false
```
Comment (mvn23) [inline source/_integrations/opentherm_gw.markdown]: Does `required` make sense for a dropdown?

File: source/_integrations/opentherm_gw.markdown [hunk 3]
```diff
@@ -60,12 +60,19 @@ The precision and floor_temperature settings that were supported in configuratio
...
+  type: float
+  default: "`0.5` for Celsius and `1.0` for Fahrenheit."
+Set Precision:
+  description: "The desired set precision for this device. Used as step size for setting temperature setpoint from the climate entity. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+  required: false
```
Comment (mvn23) [inline source/_integrations/opentherm_gw.markdown]: Same as above.

File: source/_integrations/opentherm_gw.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/opentherm_gw.markdown]: No, it does not, but there it no real option here.

This `configuration` block, should be converted into a `configuration_basic` block, which does not have all these additional (and required) fields.

Comment (Martidjen) [reply source/_integrations/opentherm_gw.markdown]: Will do. So I can lose the 'type' and 'required' fields. Should I keep the 'default' value? Or should it be just the description?

Comment (frenck) [reply source/_integrations/opentherm_gw.markdown]: Just the description, the default value is not part of the base (and not needed with UI configurations).

**Review by mvn23 — APPROVED** [review]
:+1:

**Review by frenck — APPROVED** [review]
Thanks, @Martidjen 👍

**Pull request merged**